insightfinder/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "display_name": "InsightFinder",
+  "maintainer": "support@insightfinder.com",
+  "manifest_version": "1.0.0",
+  "name": "insightfinder",
+  "metric_prefix": "insightfinder.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Integrate data from DataDog for analysis by InsightFinder",
+  "guid": "1ecaeadf-7b34-48f7-b9cb-81b8a9e4e4dc",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "InsightFinder",
+  "categories": [
+    ""
```

Comment:
Do you want to add any categories for this integration?

Reply:
@dearmore, if you would like to include a category or two, please select from the options in the following word cloud: https://docs.datadoghq.com/integrations/.

Reply:
@alexvetras @sarah-witt Thanks for the feedback, I have updated the categories. Let me know if there's anything else needed. Thanks!